core/src/replicator.rs

```diff
@@ -425,13 +424,12 @@ impl Replicator {
         let bal = client.poll_get_balance(&storage_keypair.pubkey());
         if bal.is_err() || bal.unwrap() == 0 {
             let (blockhash, _fee_calculator) = client.get_recent_blockhash().expect("blockhash");
-            //TODO the account space needs to be well defined somewhere
             let tx = system_transaction::create_account(
                 keypair,
                 &storage_keypair.pubkey(),
                 blockhash,
                 1,
-                1024 * 4,
+                1024 * 4, // TODO the account space needs to be well defined somewhere
```

Comment:
fyi https://github.com/solana-labs/solana/blob/master/programs/budget_api/src/budget_instruction.rs#L42

Reply:
These accounts have HashMaps in them and so I can't just construct a default and check its serialized size. I'll probably have to use a hardcoded size somewhere.